ush/retrieve_data.py

```diff
@@ -219,12 +220,13 @@ def fill_template(template_str, cycle_date, templates_only=False, **kwargs):
         yyyymm=cycle_date.strftime("%Y%m"),
         yyyymmdd=cycle_date.strftime("%Y%m%d"),
         yyyymmddhh=cycle_date.strftime("%Y%m%d%H"),
+        yyjjjhh=cycle_date.strftime("%y%j%H"),
```

Comment:
i believe that {jjj} was not recognized as a  variable from the dictionary when i tried {yy}{jjj}{hh}, but let me double-check. i agree it would make sense to do it the way you mention and not add the yyjjjhh entry.

Reply:
removed yyjjjhh entry here and instead used {yy}{jjj}{hh} when needed in data_locations.yaml